src/main/scala/org/clulab/wm/eidos/serialization/json/JLDDeserializer.scala

```diff
@@ -155,9 +154,11 @@ class JLDDeserializer {
     val text = (timexValue \ "text").extract[String]
     val startOffset = (timexValue \ "startOffset").extract[Int]
     val endOffset = (timexValue \ "endOffset").extract[Int]
-    // A TimeExpression, if it exists at all, must have intervals; therefore, no extractOpt here.
-    val intervals = (timexValue \ "intervals").extract[JArray].arr.map { timeIntervalValue =>
-      deserializeTimeInterval(timeIntervalValue)
+    val intervals = (timexValue \ "intervals").extractOpt[JArray] match {
+      case Some(timeIntervalsValue: JArray) => timeIntervalsValue.arr.map { timeIntervalValue: JValue =>
+        deserializeTimeInterval(timeIntervalValue)
+      }
+      case _ => Seq()
     }
```

Comment:
@kwalcock, this is the change I made. "intervals" is now optional.

Reply:
Thanks!  Reading it back in shouldn't concern anyone outside of the Clulab.